docs/pages/includes/machine-id/guides.mdx

```diff
@@ -1,6 +1,7 @@
 - [Using Machine ID with Ansible](../../machine-id/guides/ansible.mdx): How to integrate Machine ID with Ansible.
 - [Using Machine ID with Applications](../../machine-id/guides/applications.mdx): How to use Machine ID to connect automated services to applications.
 - [Using Machine ID with Databases](../../machine-id/guides/databases.mdx): How to use Machine ID to connect custom applications to your databases.
-- [Using Machine ID with GitHub Actions](../../machine-id/guides/github-actions.mdx): How to use Machine ID to to access Teleport resources from GitHub Actions.
+- [Using Machine ID with GitHub Actions](../../machine-id/guides/github-actions.mdx): How to use Machine ID to SSH into Teleport nodes from GitHub Actions.
```

Comment:
Is this a regression, or can you only use Machine ID with to SSH, not to access other resources?

Reply:
Machine ID can be used with a variety of our protocols, but this guide in specific shows using it for SSH. As I'm planning on writing guides for the other protocols, I thought it would be pertinent to modify the guide description for the existing guide to specify it covers the SSH part.